model_templates/python3_onnx_multiclass/preprocessing.py

```diff
@@ -88,7 +94,7 @@ def text_selector(X):
 text_pipeline = Pipeline(
     steps=[
         ("imputer", SimpleImputer(strategy="constant", fill_value="missing")),
-        ("tfidf", MultiColumnTfidfVectorizer(ngram_range=(1, 2))),
+        ("tfidf", TfidfVectorizer(ngram_range=(1, 2))),
```

Comment:
does this still work?

Reply:
Yes, this example uses skyserver dataset, which doesn't has only target field as text